### ISSUE TITLE
indent rule on return inside conditional

### ISSUE THREAD

**Issue by PierBover** [eslint/eslint#8046]
For some reason this doesn't throw an indentation error:
```js
if (items[i].id == id) {
return items[i];
}
```
But this (obviously) does although only in the `foo` line:
```js
if (items[i].id == id) {
var foo = 'bar';
return items[i];
}
```

![image](https://cloud.githubusercontent.com/assets/226483/22754275/a6d6172c-ee04-11e6-9926-7809d785ed7d.png)

I would expect the `return` line to throw an error since it's not indented.

I'm using this rule for indents:
```json
"indent": ["error", "tab", {"SwitchCase": 1}]
```

Here is the complete set of rules: https://gist.github.com/PierBover/992647a6fe163dfadf1cea3243c96c68

I'm using Atom 1.13.1 on macOS with linter-eslint 8.1.0 which apparently uses ESLint 3.6.0

Node: 7.0.3
NPM: 3.10.10

No parser, this is ES5 code.

**Comment by gyandeeps** [comment]
Was not able to reproduce it on the demo app: http://eslint.org/demo/

I do see an error
`4:2 - Expected indentation of 2 tabs but found 1. (indent)`

```js
/* eslint indent: [2, "tab"] */
function test() {
	if (items[i].id == id) {
	return items[i];
	}
}
```

**Comment by PierBover** [comment]
Thanks for checking.

I will report on the linter-eslint repo.
